### DISCUSSION TITLE
Streamflow routing--getting started

### DISCUSSION THREAD

**Discussion by cbblanke** [NASA-LIS/LISF#1556] [Q&A] [answered]
Is there a guide describing what to do to set up a streamflow routing model in LIS?  I see the options in the ldt.config file for HYMAP river width, height, roughness, etc.?  Is LDT going to create these?
Does LIS need to run at the same resolution as the LSM and along with the LSM?  Or can it run the streamflow by itself based on an existing LSM output?

Thanks

**Comment by agetiran** [comment, marked as answer]
Hi @cbblanke,
HyMAP is the river model in LIS used to simulate surface water dynamics (streamflow included). You can run it coupled with an LSM, or alone, if you have daily LIS outputs containing Qs and Qsb. Each way requires specific options in the lis.config file. 

Previously generated HyMAP parameters must be provided in the ldt.config file, as you mentioned above, that will be cropped and included in the lis_input.nc file. Currently, global parameters are available at 0.25 deg (maybe there's a 0.1 deg version as well), and over CONUS at 0.125 deg. If your project is at a different spatial resolution, please send me an email to my NASA address with coordinates and resolution so I can generate customized parameters and answer any additional questions you may have.

Augusto
